base/slicearray.jl

```diff
@@ -146,20 +152,29 @@ See also [`eachcol`](@ref), [`eachslice`](@ref) and [`mapslices`](@ref).
 # Example
 
 ```jldoctest
-julia> a = [1 2; 3 4]
-2×2 Matrix{Int64}:
- 1  2
- 3  4
+julia> a = [1 2 3; 40 50 60]
+2×3 Matrix{Int64}:
+  1   2   3
+ 40  50  60
```

Comment:
Changing these examples from 2x2 to 2x3 will, I hope, make it more obvious which dimension is which.